Resources/Prototypes/_Funkystation/Entities/Structures/Machines/BluespaceGas/bluespace_gas.yml

```
@@ -0,0 +1,166 @@
+      enum.PowerDeviceVisuals.Powered:
+        enum.PowerDeviceVisualLayers.Powered:
+          True: { state: on }
+          False: { state: off }
+  - type: DeviceNetwork
+    deviceNetId: Wireless
+  - type: WirelessNetworkConnection
+    range: 1000
+  - type: DeviceLinkSource
+    range: 1000
+    ports:
+      - BluespaceSender
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)